synapse/models/risk.py

```diff
@@ -313,6 +313,9 @@ def getModelDefs(self):
                     ('reporter:name', ('ou:name', {}), {
                         'doc': 'The name of the organization reporting on the mitigation.'}),
 
+                    ('mitre:attack:mitigation', ('it:mitre:attack:mitigation', {}), {
+                        'doc': 'A mapping to a Mitre ATT&CK mitigation if applicable.'}),
```

Comment:
Should this be MITRE?

Reply:
I went with how it was written on ou:campaign and ou:technique, but it's worth a go-back to see if we should change all of them.

Reply:
I'm submitting a story to review the other docs and make them consistent